app/views.py

```diff
@@ -33,3 +48,11 @@ def page_forbidden(error):
 @url.errorhandler(500)
 def page_server_error(error):
     return render_template('500.html', title='Server Error'), 500
+
+
+def convert_to_readable_time(time):
+    return time.strftime('%b/%d/%Y %a %I:%M:%S %p')
```

Comment:
@dashmug Wouldn't it be more understandable if the month is translated to its abbreviation? The above line returns:
 Apr/7/16